st2api/st2api/controllers/v1/actions.py

```diff
@@ -78,8 +78,13 @@ def __init__(self, *args, **kwargs):
 
     @request_user_has_permission(permission_type=PermissionType.ACTION_LIST)
     @jsexpose()
-    def get_all(self, **kwargs):
-        return super(ActionsController, self)._get_all(**kwargs)
+    def get_all(self, exclude_attributes=None, **kwargs):
+        if exclude_attributes:
```

Comment:
In action executions controller we validate that a user provided a valid exclude attribute (`_validate_exclude_fields`) so probably not a bad idea to refactor some of that functionality on a base controller and also do that here (and introduce a whitelist of attributes which can be excluded).